4.3/README.md

```diff
@@ -42,9 +44,17 @@ If you want to check that your data import was successful, you can use the API w
 
 The following environment variables are available for configuration:
 
-- `PBF_URL`: Which [OSM extract](#openstreetmap-data-extracts) to download and import. It cannot be used together with `PBF_PATH`. Check [https://download.geofabrik.de](https://download.geofabrik.de)
+- `PBF_URL`: Which [OSM extract](#openstreetmap-data-extracts) to download and import. It cannot be used together with `PBF_PATH`.
+  Check [https://download.geofabrik.de](https://download.geofabrik.de) 
+  Since de DL speed is restrictet at Geofabrik, for importing the full Planet there is a recommended list of Mirrors at the [OSM Wiki](https://wiki.openstreetmap.org/wiki/Planet.osm#Planet.osm_mirrors).
+  There you can find the folder /planet which contains the Planet-latest.osm.pbf
+  and mostly a /replication folder for the `REPLICATION_URL`.
 - `PBF_PATH`: Which [OSM extract](#openstreetmap-data-extracts) to import from the .pbf file inside the container. It cannot be used together with `PBF_URL`.
-- `REPLICATION_URL`: Where to get updates from. Also available from Geofabrik.
+- `REPLICATION_URL`: Where to get updates from. For exampe at Geofabrik under for example europe: https://download.geofabrik.de/europe-updates/
+                     The Europe member Countrys Update Path in in https://download.geofabrik.de/europe/
+                    if only osm.pbf files visible delete at the end of the Site URL the.html ...europe/ instad of ...europe.html          
+                     Other Map-Update paths like this example https://download.geofabrik.de/countryname-updates/ 
+- `nominatim-flatnode:`: Set the path for the Flatnode File (Suggested:/nominatim/flatnode). 
```

Comment:
That's not an environment variable. Please remove this. You can instead update the separate flatnode section.